### ISSUE TITLE
has_many and check_boxes : css issue

### ISSUE THREAD

**Issue by afriqs** [activeadmin/activeadmin#3100]
Given:
```ruby
 f.has_many :practicings, heading: nil, allow_destroy: true, new_record: true do |p|
   p.input :disciplines
end
```
![select](https://cloud.githubusercontent.com/assets/318464/2754243/b3110d94-c94f-11e3-8362-4a4ad4477f79.png)

The select box is well displayed for a habtm relationship. But given:
```ruby
 f.has_many :practicings, heading: nil, allow_destroy: true, new_record: true do |p|
   p.input :disciplines, as: :check_boxes
end
```
![check_boxes](https://cloud.githubusercontent.com/assets/318464/2754256/efaa3a3c-c94f-11e3-823b-f8ee0e61cfdb.png)

The check boxes are stuck to the left.

Note : Rails 4.0.4 and ActiveAdmin 1.0.0.pre (65aae6b68e78)

**Comment by pencilcheck** [comment]
+1 Same issue

**Comment by timoschilling** [comment]
closed by #3812
